userProfile/forms.py

```diff
@@ -25,3 +34,17 @@ def __init__(self, *args, **kwargs):
                 'class': 'form-control'
         })
 
+    def save(self):    #saving the values into the vars we initilialized before
+        photo = super(ProfileForm, self).save()     #initializing super here!!!
```

Comment:
clean these comments a bit. Comments fine to have just should make sense to people who have no idea whats up